tests/test_configuration_parser.py

```diff
@@ -252,10 +252,33 @@ def test_validate_config_experiment_id_9(self):
                 'really_really_really_long_id',
                 'experiment_dirs': []}
 
+        # Add data to `ConfigurationParser` object
+        self.parser.load_config_from_dict(data)
+        self.parser.validate_config(context='rsmsummarize')    
+
+
+    @raises(ValueError)
+    def test_validate_config_too_many_experiment_names(self):
+        data = {'summary_id': 'summary',
+                'experiment_dirs': ["dir1", "dir2", "dir3"],
+                'experiment_names': ['exp1', 'exp2', 'exp3', 'exp4']}
+
+        # Add data to `ConfigurationParser` object
+        self.parser.load_config_from_dict(data)
+        self.parser.validate_config(context='rsmsummarize')
+
+
+    @raises(ValueError)
+    def test_validate_config_too_many_experiment_names(self):
```

Comment:
This has a duplicated name. Should this be `test_validate_config_too_few_experiment_names`?